docs/supervisor.md

```diff
@@ -14,7 +14,7 @@
 | Velero Plugin for vSphere Version | vSphere Version     | Kubernetes Version                                                | vSphere CSI Driver Version | Velero Version | Velero vSphere Operator Version | Data Manager Version | vSphere Plugin Deprecated | vSphere Plugin EOL Date      |
 |-----------------------------------|---------------------|-------------------------------------------------------------------|----------------------------|----------------|---------------------------------|---------------|------------|---------------|
 | 1.4.2                             | 8.0                  | Bundled with vSphere (1.22-1.23)                                 | Bundled with vSphere       | 1.9.2          | 1.3.0                           | 1.2.0        | No         | N/A           |
-| 1.4.2                             | 7.0U3h               | Bundled with vSphere (1.22)                                      | Bundled with vSphere       | 1.9.2          | 1.3.0                           | 1.2.0        | No         | N/A           |
+| 1.4.2                             | 7.0U3h  (**Warning:** Velero vSphere Operator Pod will restart periodically )             | Bundled with vSphere (1.22)                                      | Bundled with vSphere       | 1.9.2          | 1.3.0                           | 1.2.0        | No         | N/A           |
```

Comment:
Can you add that "Velero can only be installed while the Velero vSphere Operator Pod is running."